packages/react-components/src/checkbox/src/Checkbox.css

```diff
@@ -7,7 +9,7 @@
     line-height: 1;
     max-width: 100%;
     width: max-content;
-    height: max-content;
```

Comment:
Why did you remove the max-content? Ideally we would keep this for every components